gromacs/gromacs-2022.x/colvarproxy_gromacs.cpp

```diff
@@ -47,9 +47,14 @@ void colvarproxy_gromacs::init(t_inputrec *ir, int64_t step, const gmx_mtop_t &m
 
   angstrom_value = 0.1;
 
+  // From Gnu units
+  // $ units -ts 'k' 'kJ/mol/K/avogadro'
+  // 0.0083144621
+  boltzmann_ = 0.0083144621;
```

Comment:
It could be worth using the GROMACS constant here:  see `gmx::c_boltz`

Reply:
I know we always write the number before but we could benefit from the PR

Reply:
@HubLot I couldn't the definition of `c_boltz` anywhere. Is it auto-generated? I tried guessing from what similar source files include.

Reply:
I tried but...
![image](https://user-images.githubusercontent.com/4512089/214936587-ba4c807a-722b-44dc-80fc-5ee3bfa4bb74.png)

Reply:
okay so:
  - for 2020 & 2021: `#include "gromacs/math/units.h"` with `BOLTZ` as a name (without `gmx::`)
  - for 2022: `#include "gromacs/math/units.h"` with `gmx::c_boltz`

Reply:
didn't catch they changed the constant name in 2022 and above :)

Reply:
Okay this works. Thanks!